### PR TITLE
Patches core tier

### PR DESCRIPTION
**Pull request by stephanie-rifai**
update the recipes to default to the free-tier version of the experiment file and make use of patches for the paid version. Recipes changed: locust-metrics, voting-web-app, and elasticsearch. Elastic search also now includes a kustomize.

### CODE REVIEW

File: elasticsearch/advanced/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Elasticsearch Advanced Example
+
+## Background
+This experiment builds upon our [Elasticsearch Recipe](https://github.com/redskyops/redskyops-recipes/tree/master/elasticsearch). The following parameters are patched to the experiment file:
+* data_replicas
+* client_replicas
+* client_heap_percent
+
+## Prerequisites
+In order to run this experiment you most have a paid version of Red Sky Ops. To upgrade visit our [pricing page](https://www.carbonrelay.com/pricing/).
```
Comment (jimschley) [inline elasticsearch/advanced/README.md:10]: s/most have/must have/

File: locust-metrics/advanced/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+* worker_replicas
+* redis_memory
+* redis_cpu
+
+This experiment also optimizes for two additional metrics:
+* cost
+* throughput
+
+## Prerequisites
+In order to run this experiment you most have a paid version of Red Sky Ops. To upgrade visit our [pricing page](https://www.carbonrelay.com/pricing/).
```
Comment (jimschley) [inline locust-metrics/advanced/README.md:15]: s/most have/must have/

File: voting-webapp/advanced/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+## Background
+This experiment builds upon our [Voting Web App Recipe](https://github.com/redskyops/redskyops-recipes/tree/master/voting-webapp). The following parameters are patched to the experiment file:
+* worker_memory
+* worker_replicas
+* redis_memory
+* redis_cpu
+
+
+## Prerequisites
+In order to run this experiment you most have a paid version of Red Sky Ops. To upgrade visit our [pricing page](https://www.carbonrelay.com/pricing/).
```
Comment (jimschley) [inline voting-webapp/advanced/README.md:12]: s/most have/must have/

**Review by jimschley — CHANGES_REQUESTED** [review]
Looks good. Added single line comments for typos in READMEs.

File: elasticsearch/advanced/README.md (hunk 1 above)
Comment (stephanie-rifai) [reply elasticsearch/advanced/README.md:10]:
```suggestion
In order to run this experiment you must have a paid version of Red Sky Ops. To upgrade visit our [pricing page](https://www.carbonrelay.com/pricing/).
```

File: voting-webapp/advanced/README.md (hunk 3 above)
Comment (stephanie-rifai) [reply voting-webapp/advanced/README.md:12]:
```suggestion
In order to run this experiment you must have a paid version of Red Sky Ops. To upgrade visit our [pricing page](https://www.carbonrelay.com/pricing/).
```

File: locust-metrics/advanced/README.md (hunk 2 above)
Comment (stephanie-rifai) [inline locust-metrics/advanced/README.md:15]:
```suggestion
In order to run this experiment you must have a paid version of Red Sky Ops. To upgrade visit our [pricing page](https://www.carbonrelay.com/pricing/).
```

Comment (stephanie-rifai) [reply locust-metrics/advanced/README.md]: nice catch - thanks!

**Review by jimschley — APPROVED** [review]

File: elasticsearch/experiment.yaml [hunk 4]
```diff
@@ -101,23 +86,21 @@ spec:
         - name: cluster.name
           value: rally-demo
         - name: client.replicas
-          value: "{{ .Values.client_replicas }}"
+          value: "2"
```
Comment (Ofer-Idan) [inline elasticsearch/experiment.yaml:89]: If we're not tuning replicas, should this just move from the experiment.yaml to the application manifests?

**Pull request merged**